lib/neo4j/driver/exceptions/security_exception.rb

```diff
@@ -1,10 +0,0 @@
-# frozen_string_literal: true
-
-module Neo4j
-  module Driver
-    module Exceptions
-      class ProtocolException < Neo4jException
```

Comment:
Same class is defined in protocol_exception.rb in same dir [here](https://github.com/neo4jrb/neo4j-ruby-driver/blob/master/lib/neo4j/driver/exceptions/protocol_exception.rb).

I have to remove this one because the file name didn't match the convention of classname and zeitwerk throws the error. Was wondering if this classname was supposed to be SecurityException instead?

Reply:
this is a mistake it should read `SecurityException`

Reply:
done, renamed class name to SecurityException, instead of removing the whole file